package.json

```diff
@@ -20,7 +20,7 @@
         "typecheck": "tsc --noEmit",
         "build": "rm -rf dist && ENV=production webpack -p --progress",
         "test": "jest",
-        "testcheck": "npm-run-all --parallel typecheck test",
+        "testcheck": "yarn test && yarn typecheck && yarn prettify:check",
```

Comment:
Why not run in parallel? Seems that helps speed up build/deploy?

Reply:
It garbles the output since they all write to stdout. Some of them also have the fancy loading indicators that completely overwrite messages...

If we only care about a final pass/fail then we can do that, `npm-run-all` will have a non-zero exit code if a single or more processes have a non-zero exit code.

But if we care about seeing what the errors are, then it's better to do it this way.